examples/cpp-simple/server.cc

```diff
@@ -109,6 +133,7 @@ int main() {
       if (!status.ok()) {
         std::cerr << "Could not run Shutdown():" << status.error_message()
                   << ". Exiting!\n";
+        join_threads(std::move(health), std::move(watch));
```

Comment:
I think this function call (and the function itself) were meant to be removed?

Otherwise, I think this change looks good. Thanks for the fix and the updates!